vortex-array/src/array/varbin/builder.rs

```diff
@@ -66,6 +66,12 @@ impl<O: NativePType + PrimInt> VarBinBuilder<O> {
         self.validity.append_null();
     }
 
+    #[inline]
+    pub fn push_n_nulls(&mut self, n: usize) {
```

Comment:
Use Builder::append_nulls instead. Maybe we should rename hte other functions to "append"